jdaviz/configs/default/plugins/plot_options/plot_options.vue

```diff
@@ -39,6 +39,31 @@
       :hint="viewer_multiselect ? 'Select viewers to set options simultaneously' : 'Select the viewer to set options.'"
     />
 
+    <v-row v-if="viewer_selected === 'spectrum-viewer' || (viewer_selected[0] === 'spectrum-viewer' && viewer_selected.length === 1)">
+      <v-expansion-panels popout>
+        <v-expansion-panel>
+          <v-expansion-panel-header v-slot="{ open }">
+            <span style="padding: 6px">Viewer bounds</span>
+          </v-expansion-panel-header>
+          <v-expansion-panel-content class="plugin-expansion-panel-content">
+            <v-row v-for="item in spectrum_viewer_limits">
+              <v-text-field
+              ref="item"
+              :label="item.label"
+              v-model.number="item.value"
+              type="number"
+              step="0.1"
+              :suffix="item.unit"
+              ></v-text-field>
+            </v-row>
+            <v-row>
+              <plugin-action-button :results_isolated_to_plugin="true/false" @click="apply_updated_bounds">Apply Updated Bounds</plugin-action-button>

```

Comment:
```suggestion
              <plugin-action-button :results_isolated_to_plugin="false" @click="apply_updated_bounds">Apply Updated Bounds</plugin-action-button>

```

Reply:
That dangerous moment between copying and pasting when I forget I need to change something 😆